app/Http/Controllers/Api/ApiController.php

```diff
@@ -64,12 +65,18 @@ public function item($data, $code = 200, $meta = [], $transformer = null)
      * @param  array  $meta
      * @return \Illuminate\Http\JsonResponse
      */
-    public function transform($data, $code = 200, $meta = [], $include = null)
+    public function transform($data, $code = 200, $meta = [], $include = null, $endpoint = null)
     {
         $data->setMeta($meta);
 
         $manager = new Manager;
-        $manager->setSerializer(new DataArraySerializer);
+
+        // If this is a request to /reportbacks, send to PaginatorForPhoenixAshesGallery to match response expected for pagination in Phoenix Ashes Gallery.
+        if ($endpoint === 'reportbacks') {
+            $manager->setSerializer(new PaginatorForPhoenixAshesGallery);
```

Comment:
Hmm, I'd love to inline some of this into the reportback controller specifically so we don't add complexity elsewhere. Maybe we just override this method in that controller?